assets/script/script.js

```diff
@@ -8,26 +11,91 @@ fetch("https://api.football-data.org/v2/competitions/PL/teams", {
     return response.json();
   })
   .then(function (data) {
-    footballData = data;
+    teamData = data;
   })
   .then(renderCrests)
-  .then(renderTeamInfo);
+  .then(fillTagsArray);
 
+// get data for league standings
+fetch("https://api.football-data.org/v2/competitions/PL/standings", {
```

Comment:
Should this `fetch` be in its own function? Will you ever have cause for calling this function repeatedly?